src/lib/Components/Inbox/Conversations/Composer.tsx

```diff
@@ -42,6 +44,9 @@ interface State {
   text: string
 }
 
+const track: Track<Props, State, Schema.Entity> = _track
```

Comment:
Maybe we should make `Schema.Entity` the default schema, seeing as you seem to be defining that in many places?

Reply:
This still needs to be done?

Reply:
So, I don't quite follow... All the interfaces are non default. Do we merge `Entity` with `Global`? Or export the `Entity` as `default`? And then on import, we just use `Schema`? Or can we just import the `Schema` as a whole?

Reply:
The idea is to [here](https://github.com/artsy/emission/blob/08a9039abb7510e1209f996b458f935ae1330131/src/lib/utils/track.ts#L174) change to:

```ts
T extends Schema.Global = Schema.Entity
```

This way the above `_track` cast can be liimted to:

```ts
const track: Track<Props, State> = _track
```